ClientApp/src/components/ListingComp.js

```diff
@@ -63,7 +63,7 @@ export const ListingComp = (props) => {
     <div>
       <div className="row" style={{ borderStyle: "solid", borderWidth: "2px", marginTop: "10px" }}>
         <div>
-          <a href={'/details/' + props.id} target="_blank" rel ="noopener noreferrer" >      
+          <a href={'/details/' + props.id} target="_blank" rel="noopener noreferrer" >
```

Comment:
I think the noreferrer keyword alone is enough to secure the client from the possibility of an attack since it already implies the noopener keyword behavior.

Reply:
when I remove noopener keyword linter throws a warning:
![image](https://user-images.githubusercontent.com/90030313/140311046-b4847509-a196-459e-9c5c-87d17122d844.png)

So IMO it's best to leave it as it is.